cmd/frontend/internal/app/ui/handlers.go

```diff
@@ -296,14 +295,7 @@ func serveHome(w http.ResponseWriter, r *http.Request) error {
 		return nil // request was handled
 	}
 
-	if envvar.SourcegraphDotComMode() && !actor.FromContext(r.Context()).IsAuthenticated() && !strings.Contains(r.UserAgent(), "Cookiebot") {
-		// The user is not signed in and tried to access Sourcegraph.com.  Redirect to
-		// about.sourcegraph.com so they see general info page.
-		// Don't redirect Cookiebot so it can scan the website without authentication.
-		http.Redirect(w, r, (&url.URL{Scheme: aboutRedirectScheme, Host: aboutRedirectHost}).String(), http.StatusTemporaryRedirect)
-		return nil
-	}
-	// On non-Sourcegraph.com instances, there is no separate homepage, so redirect to /search.
+	// Homepage redirects to /search
```

Comment:
Ed-van-nit speaking:

```suggestion
	// Homepage redirects to /search.
```